### ISSUE TITLE
Allow specification of array keys within various responses

### ISSUE THREAD

**Issue by ultrono** [f9webltd/laravel-api-response-helpers#18]
For methods such as [`respondUnAuthenticated`](https://github.com/f9webltd/laravel-api-response-helpers/blob/master/src/ApiResponseHelpers.php#L60-L66) add a second optional argument to set the (in this case), 'error' key.

**Comment by kingyyy** [comment]
same

**Comment by kingyyy** [comment]
yes

**Comment by kingyyy** [comment]
yes

**Comment by Hesesses** [comment]
I agree, just error message is not enough. There should be both, human readable message and an error key
